src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -245,18 +243,23 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
       postalCode: usOrCanada && validatePresence(postalCode),
       phoneNumber: validatePresence(phoneNumber),
     }
+    return {
+      errors,
+      hasErrors: Object.keys(errors).filter(key => errors[key]).length > 0,
```

Comment:
#minor, but would be nice to separate a complex operation from object assignment, i.e., 

```js
const hasErrors = Object.keys(errors).filter(key => errors[key]).length > 0,

return {
  errors,
  hasErrors
}
```